linters/psscriptanalyzer/plugin.yaml

```diff
@@ -0,0 +1,43 @@
+version: 0.1
+downloads:
+  - name: psscriptanalyzer
+    downloads:
+      - os:
+          linux: linux
+          windows: windows
+          macos: macos
+        url: https://github.com/PowerShell/PSScriptAnalyzer/releases/download/${version}/PSScriptAnalyzer.${version}.nupkg
+tools:
+  definitions:
+    - name: psscriptanalyzer
+      download: psscriptanalyzer
+      shims: [PSScriptAnalyzer.psd1]
+      known_good_version: 1.21.0
+lint:
+  definitions:
+    - name: psscriptanalyzer
+      files: [powershell]
+      tools: [psscriptanalyzer, pwsh]
+      commands:
+        - name: Invoke-ScriptAnalyzer
+          run: |
+            pwsh -f ${cwd}/lint.ps1 -ModuleDir ${linter} -FilePath ${target} -OutputPath ${tmpfile}
```

Comment:
Can you inline this? Otherwise yaml actually treats there as being an implicit new lint at the end. I think that is the source of the test failure in CI.

Reply:
I removed the pipe but then (I assume) Prettier went and pushed the statement onto the next line anyway. Hopefully that is okay?

Reply:
I think that should be fine. The tests matched my observations locally so if those pass we should be good

Reply:
Other changes I made made this line short enough now, anyway.